Examples/Algorithms/TrackFinding/src/TrackFindingOptions.cpp

```diff
@@ -24,6 +25,7 @@ void ActsExamples::Options::addTrackFindingOptions(
   opt("ckf-selection-nmax", value<size_t>()->default_value(10),
       "Global criteria of maximum number of measurement candidates on a "
       "surface for CKF measurement selection");
+  opt("ckf-truth-seeds", bool_switch(), "Use truth seeds for steering CKF");
```

Comment:
This is a specific option for one tool to select which algorithms to add and not something that configures a specific algorithm. The option should probably be defined not here but directly in the executable.

Reply:
That's a good point. It has been moved to the executable.